pkg/adsc/delta.go

```diff
@@ -271,12 +292,28 @@ func (c *Client) Run(ctx context.Context) error {
 	c.sendNodeMeta.Store(true)
 	c.xdsClient = xdsClient
 	go c.handleRecv()
+	// TODO: if reconnect, may add subscribe info to the request.
```

Comment:
We should take subscribed resource names when reconnect

Reply:
For the envoy type, we need to traverse the `tree`; for the mcp type we need to traverse the `config store`, which is not natively accessible to delta client.